ginga/examples/configs/general.cfg

```diff
@@ -72,3 +72,8 @@ icc_working_profile = "sRGB"
 # acceptable values are: 'off', 'on' or 'auto' (as needed)
 # NOTE: overridden by channel setting of the same name, if any
 scrollbars = 'auto'
+
+# Set this to True to have Ginga remember the size and position of the
+# last session. This creates/loads ~/.ginga/layout file.
+# Setting this to False would cause Ginga to ignore the file, if exists.
```

Comment:
I would add to the comment: "If you experience trouble starting up ginga you may need to remove the ~/.ginga/layout file and restart the program."

Reply:
Done! 😄